Tests/NIOExtrasTests/XCTestManifests.swift

```diff
@@ -0,0 +1,195 @@
+#if !canImport(ObjectiveC)
```

Comment:
We have a script checked into this repository that generates the tests, at `scripts/generate_linux_tests.rb`. Can you discard these manifests and use that script instead?